README.md

```diff
@@ -44,8 +44,20 @@ The config section contains meta-data about the template. All of these attribute
 * name - The unique template name. All lower-case and only the characters a-z, 0-9 and the dash.
 * label - The human-readable template name, used for display.
 * description - An English description of the template, used for display.
+* category - ["blogging" | "hosting" | "infrastructure" | "email" | "e-commerce" | "support" ]
```

Comment:
I'd simply use `ecommerce`.

I'm still not 100% convinced by "hosting" vs "infrastructure", but I can't find a better alternative now. It's confusing to me because Heroku could easily be considered "hosting".

office-365 is also better suited to "productivity", but I'm not sure which other services will fall into this category today. What do you think?